src/test/java/reposense/git/GitLsTreeTest.java

```diff
@@ -0,0 +1,23 @@
+package reposense.git;
+
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.Test;
+
+import reposense.template.GitTestTemplate;
+import reposense.util.TestUtil;
+
+public class GitLsTreeTest extends GitTestTemplate {
+
+    @Before
+    public void before() {
+        // Skips GitLsTreeTest on Unix systems
+        Assume.assumeFalse(!TestUtil.isWindows());
+    }
+
+    @Test(expected = InvalidFilePathException.class)
+    public void repo_containsInvalidFilePaths_runtimeException() throws InvalidFilePathException {
+        config.setBranch("391-invalid-filepaths");
+        GitLsTree.validateFilePaths(config);
```

Comment:
Hmmm, let's also add a 'negative' example, we run the `validateFilePaths` on the `master` branch as well?